environment.yaml

```diff
@@ -14,6 +14,7 @@ dependencies:
 - networkx>=1.10
 - pyomo>=5.7.0,<6.6.2
 - cartopy>=0.16
+- geopandas>=0.9
```

Comment:
Is there any downside to having `geopandas` as compulsory dependency?

Reply:
not sure, but since we are already having cartopy, I'd say that's a minor addition